### DISCUSSION TITLE
TOC issue

### DISCUSSION THREAD

**Discussion by 0xRyuzak1** [cotes2020/jekyll-theme-chirpy#1345] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hello Everyone

Please i need some help i found that when i write a new post the H1 heading are not listed on the table of content of the page so can you help me to fix this issue .

### Operations you have already tried

i can't find solution to this issue

**Comment by tomasApo** [comment]
By table of contents I'm going to assume  you mean the home tab. Have you created a file in your `/_posts/` directory? 

If not use the naming convention of 2023-10-29-NameOfProject.md

**Reply by 0xRyuzak1** [reply]
The Toc is the table of content which will appears on the right side when you open a post

**Reply by tomasApo** [reply]
got it, are H2 or H3 headings showing up on TOC?

**Reply by 0xRyuzak1** [reply]
yes starting from h2 to all they showing but not h1

**Comment by NichtsHsu** [comment, marked as answer]
[Markdownlint rule MD025](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md025---multiple-top-level-headers-in-the-same-document) recommends to use only single H1 header as the title. Because jekyll automatically generates H1 level titles for your articles, you'd better start with H2 headers.

**Reply by 0xRyuzak1** [reply]
Thank you very much @NichtsHsu this is really helped .
